frontend/src/metabase/containers/ItemPicker/ItemPicker.tsx

```diff
@@ -86,7 +87,7 @@ function ItemPicker<TId>({
   className,
   showSearch = true,
   showScroll = true,
-  showOnlyPersonalCollections = false,
+  filterPersonalCollections,
```

Comment:
We can remove the default value because the default is now `undefined` means we'll show everything from both public collections and personal collections.